### ISSUE TITLE
UI's `pnpm codegen` doesn't work on Windows

### ISSUE THREAD

**Issue by mmurto** [eclipse-apoapsis/ort-server#231]
As the `codegen` script for UI includes environment variables, it doesn't work on windows.

https://github.com/eclipse-apoapsis/ort-server/blob/523cacca41140eef4eac8ec3a5b70d8ed134cbaf/ui/package.json#L12

```console
$ pnpm codegen

> ui@0.0.0 codegen C:\Users\mikkom\IdeaProjects\ort-server\ui
> rm -fr src/api && openapi-rq -i http://localhost:${ORT_SERVER_CORE_PORT:-8080}/swagger-ui/api.json -o src/api --format prettier

node:internal/url:796
    this.#updateContext(bindingUrl.parse(input, base));
                                   ^

TypeError: Invalid URL
    at new URL (node:internal/url:796:36)
    at Object.resolve (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@apidevtools+json-schema-ref-parser@11.6.1\node_modules\@apidevtools\json-schema-ref-parser\dist\lib\util\url.js:54:25)
    at $RefParser.parse (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@apidevtools+json-schema-ref-parser@11.6.1\node_modules\@apidevtools\json-schema-ref-parser\dist\lib\index.js:101:25)
    at $RefParser.resolve (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@apidevtools+json-schema-ref-parser@11.6.1\node_modules\@apidevtools\json-schema-ref-parser\dist\lib\index.js:145:24)
    at $RefParser.bundle (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@apidevtools+json-schema-ref-parser@11.6.1\node_modules\@apidevtools\json-schema-ref-parser\dist\lib\index.js:165:24)
    at Object.bundle (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@apidevtools+json-schema-ref-parser@11.6.1\node_modules\@apidevtools\json-schema-ref-parser\dist\lib\index.js:160:32)
    at C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@hey-api+openapi-ts@0.43.0_typescript@5.4.5\node_modules\@hey-api\openapi-ts\dist\node\index.cjs:1:147449
    at exports.createClient (C:\Users\mikkom\Code\openapi-react-query-codegen\node_modules\.pnpm\@hey-api+openapi-ts@0.43.0_typescript@5.4.5\node_modules\@hey-api\openapi-ts\dist\node\index.cjs:1:147465)
    at async generate (file:///C:/Users/mikkom/Code/openapi-react-query-codegen/dist/generate.mjs:35:5)
    at async setupProgram (file:///C:/Users/mikkom/Code/openapi-react-query-codegen/dist/cli.mjs:39:5) {
  code: 'ERR_INVALID_URL',
  input: 'http://localhost:${ORT_SERVER_CORE_PORT:-8080}/swagger-ui/api.json',     
  base: 'c:/Users/mikkom/IdeaProjects/ort-server/ui/'
}

Node.js v20.12.0
 ELIFECYCLE  Command failed with exit code 1.

```

**Comment by sschuberth** [comment]
> As the `codegen` script for UI includes environment variables, it doesn't work on windows.

Yeah, I was aware that it wouldn't work in native Windows when I added this, but I thought you guys are always using WSL anyway 😬

The proper solution probably is to use [cross-env](https://www.npmjs.com/package/cross-env) instead.

**Comment by sschuberth** [comment]
> The proper solution probably is to use [cross-env](https://www.npmjs.com/package/cross-env) instead.

However, unfortunately `cross-env` does not seem to be able to [handle default values](https://github.com/kentcdodds/cross-env/issues/140).

**Comment by mnonnenmacher** [comment]
Fixed by #277 (don't know why autoclose didn't work in this case).
